ground/gcs/src/plugins/boards_dronin/seppuku.ui

```diff
@@ -576,6 +576,22 @@ margin: 2px 0 2px;</string>
                 </property>
                </widget>
               </item>
+              <item row="1" column="0" colspan="2">
```

Comment:
Ahh, have this done (and dynamic so it goes away when you have receiver disabled. Few other commits at home that need pushed too IIRC but I didn't here back from you the other day. ;)